db/migrate/20161207220901_add_content_to_resources.rb

```diff
@@ -0,0 +1,5 @@
+class AddContentToResources < ActiveRecord::Migration[5.0]
+  def change
+    add_column :resources, :content, :jsonb, null: false, default: {}
```

Comment:
Out of curiosity, what is the motivation for using a `:jsonb` field vs. a large text field?